bcda/cli.go

```diff
@@ -200,13 +231,23 @@ func importCCLF9(fileMetadata cclfFileMetadata) error {
 	for sc.Scan() {
 		b := sc.Bytes()
 		if len(bytes.TrimSpace(b)) > 0 {
-			fmt.Printf("\nXREF: %s\n", b[0:1])
-			fmt.Printf("Current identifier: %s\n", b[currIDStart:currIDEnd])
-			fmt.Printf("Previous identifier: %s\n", b[prevIDStart:prevIDEnd])
-			fmt.Printf("Previous identifier effective date: %s\n", b[prevIDEffDateStart:prevIDEffDateEnd])
-			fmt.Printf("Previous identifier obsolete date: %s\n", b[prevIDObsDateStart:prevIDObsDateEnd])
+
+			cclf9 := models.CCLF9{
```

Comment:
I wonder if there's something we can call this model that's more descriptive than `CCLF9`. That sounds like it refers to the file instead of its contents. The only idea I have is `CCLFBeneficiaryXRef` but there is likely a better name than that. 🤔

Reply:
right now I'm a keep it as CCLF9 for the name. This pr has been sitting here for a hot second, so I'd like to think if this was a real concern it would have popped up before now.